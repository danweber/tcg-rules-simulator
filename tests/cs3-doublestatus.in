VERSION:2

# multiple boosts
FIELD1:Plain,DW1-Van
HAND1:CS3-Wizard
GAME:P1 T3 MAIN 6
DUMP:DP,KEYWORDS

CMD:PLAY CS3-Wizard
STEP:AUTOANSWER:CAPTURE
DUMP:DP,KEYWORDS

EXIT