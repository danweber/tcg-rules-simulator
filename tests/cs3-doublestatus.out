FIELD1:Plain,DW1-Vanillamon,5000DP
FIELD2:
GAME:P1 T3 MAIN 6
CAPTURE:Target VanillaMon 1
CAPTURE:Target Wizard Hawk 2
FIELD1:Plain,DW1-Vanillamon,7000DP,SECURITY ATTACK Yyy,CS3-Wizardhawk,5500DP,LINK [-+]\D
FIELD2:
GAME:P1 T3 MAIN 0
